### PR TITLE
feat: remove productivity

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

Removes any display of a delegates productivity.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

**Comment by ghost** [comment]
The [ci/circleci: test-node-10](https://circleci.com/gh/dated/ark-explorer/1167?utm_campaign=vcs-integration-link&amp;utm_medium=referral&amp;utm_source=github-build-link) job is failing as of [d57793b10c128e7033e125fe5ee3cbb24c7913ae](https://github.com/ArkEcosystem/explorer/commit/d57793b10c128e7033e125fe5ee3cbb24c7913ae). Please [review the logs for more information](https://circleci.com/gh/dated/ark-explorer/1167?utm_campaign=vcs-integration-link&amp;utm_medium=referral&amp;utm_source=github-build-link).

_Once you've pushed the fixes, the build will automatically re-run. Thanks!_

**Comment by faustbrian** [comment]
@dated conflicts, will merge this once 2.3 hits mainnet.

**Comment by faustbrian** [comment]
@dated conflicts

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 3. _It will earn you $25 USD._

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (faustbrian) [inline app/ViewModels/Concerns/Wallet/CanForge.php]:
```suggestion
    public function hasForged(): bool
```

Keep those method names in line with the `Slot` class.

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (faustbrian) [inline app/ViewModels/Concerns/Wallet/CanForge.php]: Return early if `$this->hasForged()` is false. This is difficult to read and can result in false branch coverage reports.

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (faustbrian) [inline app/ViewModels/Concerns/Wallet/CanForge.php]:
```suggestion
    public function justMissed(): bool
```

Keep those method names in line with the `Slot` class.

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (faustbrian) [inline app/ViewModels/Concerns/Wallet/CanForge.php]:
```suggestion
    public function keepsMissing(): bool
```

Keep those method names in line with the `Slot` class.

File: resources/views/components/tables/mobile/delegates/active.blade.php
Comment (faustbrian) [inline resources/views/components/tables/mobile/delegates/active.blade.php]:
```suggestion
                    :warning="$delegate->justMissed()"
```

File: resources/views/components/tables/mobile/delegates/active.blade.php
Comment (faustbrian) [inline resources/views/components/tables/mobile/delegates/active.blade.php]:
```suggestion
                    :danger="$delegate->keepsMissing()"
```

File: resources/views/components/tables/desktop/delegates/active.blade.php
Comment (faustbrian) [inline resources/views/components/tables/desktop/delegates/active.blade.php]:
```suggestion
                    :warning="$delegate->justMissed()"
```

File: resources/views/components/tables/desktop/delegates/active.blade.php
Comment (faustbrian) [inline resources/views/components/tables/desktop/delegates/active.blade.php]:
```suggestion
                    :danger="$delegate->keepsMissing()"
```

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (ItsANameToo) [inline app/ViewModels/Concerns/Wallet/CanForge.php]: no need for `=== true` if the array only contains true/false

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (ItsANameToo) [inline app/ViewModels/Concerns/Wallet/CanForge.php]: we don't really care about random missed blocks, only if the last one is missed = yellow or if the last 2 are missed = red

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (ItsANameToo) [inline app/ViewModels/Concerns/Wallet/CanForge.php]: isn't `$this->hasForged()` always `false` here? as you have an `if` at the top that returns if `hasForged` is `true`

**Comment by ItsANameToo** [comment]
@alfonsobries conflicts

**Comment by alfonsobries** [comment]
@samharperpittam can you check again I suspect that was related to this branch not up to date with develop?

**Comment by samharperpittam** [comment]
> @samharperpittam can you check again I suspect that was related to this branch not up to date with develop?

You are correct. It'd now showing the highlighted states. I'll continue to test. Thanks @alfonsobries !

**Comment by ItsANameToo** [comment]
can you check the failing test @alfonsobries ? otherwise I'll do it tomorrow morning

File: app/ViewModels/Concerns/Wallet/CanForge.php
Comment (ItsANameToo) [inline app/ViewModels/Concerns/Wallet/CanForge.php:75]: I'll add a card for this todo, although in current implementation it should work as intended due to it not being shown until after the round has concluded

**Pull request merged**
